test/e2e/specs/linksConsolErros.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { ConsoleMessage, Page, expect } from '@playwright/test';
+import { vpTest } from '../fixtures/vpTest';
+import { LINKS } from '../testData/pageLinksData';
+
+/**
+ * Console error test generated by LINKS object array data.
+ */
+for (const link of LINKS) {
+    vpTest(`Test console errors on link ${link.name}`, async ({ page, consoleErrors, vpExamples }) => {
+        await vpExamples.clickLinkByName(link.name);
+        await waitForPageToLoadWithTimeout(page, 5000);
+        expect(page.url()).toContain(link.endpoint);
+        handleCommonBrowsersErrors(link.name, consoleErrors);
+    });
+}
+/**
+ * Testing number of links in page.
+ */
+vpTest('Link count test', async ({ page }) => {
+    const expectedNumberOfLinks = 37;
+    const numberOfLinks = await page.getByRole('link').count();
+    expect(numberOfLinks).toBe(expectedNumberOfLinks);
+});
+
+/**
+ * Helper function to handle common browser errors.
+ */
+function handleCommonBrowsersErrors(linkName: string, consoleErrors: ConsoleMessage[]) {
```

Comment:
why this and other functions are in test file?

Reply:
The functions related to this specific test. In case in the future we'll need it to other test I will move it to generic file.